src/content/release/breaking-changes/popscope-with-result.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Generic types in PopScope
+description: >
+  Added a generic type in PopScope class and updated the onPopInvoked
+  function signature.
+---
+
+## Summary
+
+Added a generic type in [`PopScope`][] class and updated the [`onPopInvoked`][]
+function signature to take in an additional result as an position parameter.
+
+## Context
+
+Previously, PopScope hadn't haven a way to access the pop result when `onPopInvoked`
+is called. To add support for accessing pop result, the generic type is added to
+PopScope class so that the `onPopInvoked` can access the type-safe result.
```

Comment:
```suggestion
Previously, `PopScope` didn't have a way to access the pop result when `onPopInvoked`
was called. The generic type is added to the `PopScope` class
so that `onPopInvoked` can access the type-safe result.
```